src/exported-image-egl.cpp

```diff
@@ -26,7 +26,7 @@
 #include "view-backend-exportable-fdo-egl-private.h"
 #include "ws.h"
 #include <cassert>
-#include <wpe-fdo/exported-image-egl.h>
+#include <wpe/exported-image-egl.h>
```

Comment:
While at it, could we also change this to use `#include "..."` for consistency with the rest of inclusions of headers which are part of the source tree?